lms/views/lti/basic_launch.py

```diff
@@ -219,9 +219,7 @@ def _configure_and_show_document(self):
         The configuration  could be because we are creating this assignment
         for the first time or from an edit.
         """
-        extra = {}
```

Comment:
I reckon this was a bad decision, but given the name `extra` seemed like the most sensible.

This version seems more explicit about what values to expect in the JSON. In any case I might be overthinking this as `group_set_id` is the only value in extra for all assignments.

We could argue that a better option could be to have `group_set_id` as a column in the assignment table.